packages/ocaml-solidity-alcotest/ocaml-solidity-alcotest.0.1.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+synopsis: "The ocaml-solidity project"
+description: """
+
+Ocaml-solidity provides a Solidity parser and typechecker
+"""
+authors: [
+  "David Declerck <david.declerck@ocamlpro.com>"
+  "Steven De Oliveira <steven.de-oliveira@ocamlpro.com>"
+]
+maintainer: [
+  "David Declerck <david.declerck@ocamlpro.com>"
+  "Steven De Oliveira <steven.de-oliveira@ocamlpro.com>"
+]
+homepage: "https://github.com/OcamlPro/ocaml-solidity"
+doc: "https://OCamlPro.github.io/ocaml-solidity/sphinx"
+bug-reports: "https://github.com/OCamlPro/ocaml-solidity/issues"
+dev-repo: "git+git+https://github.com/OcamlPro/ocaml-solidity.git"
+tags: "org:OCamlPro"
+build: [
+  ["dune" "subst"] {dev}
+  ["sh" "-c" "./scripts/before.sh build '%{name}%'"]
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["sh" "-c" "./scripts/after.sh build '%{name}%'"]
+]
+install: [
+  ["sh" "-c" "./scripts/before.sh install '%{name}%'"]
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.6.0"}
+  "ocaml-solidity-parser" {}
```

Comment:
do you want these batched packages to be `= version` so that they are all installed together? Since these are new packages, it won't matter now, but it will begin to as you issue new versions.